examples/asr/notebooks/1_ASR_tutorial_using_NeMo.ipynb

```diff
@@ -18,9 +18,7 @@
     "# If you're using Google Colab and not running locally, run this cell.\n",
     "!pip install wget\n",
     "!apt-get install sox\n",
-    "!pip install git+https://github.com/NVIDIA/apex.git\n",
-    "!pip install nemo-toolkit\n",
-    "!pip install nemo-asr\n",
+    "!pip install nemo_toolkit[asr]==0.10.0b10\n",
```

Comment:
should we pin this?

Reply:
Seems reasonable. @okuchaiev any reason not to?

Reply:
I'll pin this (in a separate commit) to a particular released version once it is out